.github/workflows/tests.yml

```diff
@@ -22,7 +22,7 @@ jobs:
         - ["3.9",   "py39"]
         - ["3.10",  "py310"]
         - ["pypy-3.9", "pypy3"]
-        - ["3.10",  "docs"]
+#        - ["3.10",  "docs"]  # disable as readthedocs builds it
         - ["3.11",  "py311"]
```

Comment:
```suggestion
#        - ["3.11",  "py311"]  # The commented Python version should be included in `.github/workflows/tests.yml`.
```

Reply:
I do  not understand this one... the py311 tox environment should be tested.

Reply:
Same. This is fine.